src/qibo_cloud_backends/aws_client.py

```diff
@@ -0,0 +1,157 @@
+    error_message = '\n'.join(error_messages)
+    print(f'The required module is not installed\n\n{error_message}')
+else:
+    from braket.aws import AwsDevice, AwsQuantumTask
+    from braket.circuits import Circuit, Gate, observables
+    from braket.circuits import Circuit
+    # from braket.device_schema import DeviceActionType
+    from braket.devices import Devices, LocalSimulator
+    # from braket.parametric import FreeParameter
+    from braket.ir.openqasm import Program
+    AWS_REQUIREMENTS = True
+    print('Modules are imported.')
+
+if AWS_REQUIREMENTS:
+    class AWSClientBackend(NumpyBackend):
+        """Backend for the remote execution of AWS circuits on the AWS backends.
+
+        Args:
+            device (str): The AWS platform. Defaults to `"LocalSimulator()"`.
+        """
+
+        def __init__(self, device=None):
+            super().__init__()
+
+            if device is None:
+                self.device = LocalSimulator()
+            # self.device = device
+            self.name = "aws"
+
+        def remove_qelib1_inc(self, qasm_string):
+            # Remove the "include "qelib1.inc";\n" line
+            modified_code = re.sub(r'include\s+"qelib1.inc";\n', '', qasm_string)
+            return modified_code
+
+        def qasm_convert_gates(self, qasm_code):
+            # To replace the notation for certain gates in OpenQASM
+            # To add more gates if necessary
+            dict = {
+                "id": "i",
+                "cx": "cnot",
+                "sx": "v",
+                "sdg": "si",
+                "tdg": "ti",
+            }
+            lines = qasm_code.split('\n')
+            modified_code = ""
+            for line in lines:
+                for key in dict:
+                    if key in line:
+                        line = line.replace(key, dict[key])
+                        break
+                modified_code += line + '\n'
+            return modified_code
+
+        def insert_verbatim_box(self, qasm_code):
+            lines = qasm_code.split('\n')
+            modified_code = ""
+        
+            register_index = -1
+            measure_index = -1
+            
+            # This loop is to find the line indices to start and close the verbatim box.
+            for i, line in enumerate(lines):
+                if "creg register" in line:
+                    register_index = i
+                if "measure" in line and measure_index == -1:
+                    measure_index = i
+        
+            # Use previously found indices to start and close the verbatim box.
+            for i, line in enumerate(lines):
+                if i == register_index:
+                    modified_code += line + "\n#pragma braket verbatim\nbox{\n"
+                elif i == measure_index:
+                    modified_code += "}\n" + line + "\n"
+                else:
+                    modified_code += line + '\n'
+        
+            return modified_code
```

Comment:
I've managed to get the Braket circuit from an OpenQASM string by doing this:
``` python
# To avoid clashing with the Qibo circuit, which is also called Circuit,
# import braket's Circuit as braket_Circuit
from braket.circuits import Circuit as braket_Circuit 

# suppose circuit_qasm is the OpenQASM string
braket_circuit = braket_Circuit.from_ir(circuit_qasm) 
```

Then we can use the `add_verbatim_box` function instead of replacing the strings in `insert_verbatim_box`
i.e. 
``` python
if verbatim_circuit:
    braket_circuit = braket_Circuit().add_verbatim_box(braket_circuit)
```